DependencyInjection/FOSRestExtension.php

```diff
@@ -37,6 +40,29 @@ public function prepend(ContainerBuilder $container)
         if ($config['view']['view_response_listener']['enabled']) {
             $container->prependExtensionConfig('sensio_framework_extra', ['view' => ['annotations' => false]]);
         }
+
+        $bundles = $container->getParameter('kernel.bundles');
+        if (isset($bundles['WebProfilerBundle'])) {
+            $profilerConfigs = $container->getExtensionConfig('web_profiler');
+            $profilerConfig = $this->processConfiguration(new ProfilerConfiguration(), $profilerConfigs);
+
+            if ($profilerConfig['toolbar'] || $profilerConfig['intercept_redirects']) {
+                $path = '_profiler';
+                if ($profilerConfig['toolbar']) {
+                    $path .= '|_wdt';
+                }
+
+                $this->formatListenerDefaultRules[] = [
```

Comment:
This is already like that https://github.com/FriendsOfSymfony/FOSRestBundle/pull/1180/files#diff-ffb88a00253e285e14e0d32ef5423cd1R192.